### ISSUE TITLE
bug while typing any "-" when creating a enrollment

### ISSUE THREAD

**Issue by sotha1** [Wavesonics/hammer-editor#245]
https://github.com/Wavesonics/hammer-editor/assets/149703967/b4a97ffd-b387-432b-a736-c56df636006b

**Comment by Wavesonics** [comment]
right, I need some messaging there about invalid characters, thanks for the report!

In the future I want to be able to encode more characters, but for now they have to be filename safe.

**Comment by sotha1** [comment]
ok, that's good!

**Comment by Wavesonics** [comment]
this has proper error messaging around it now. In the future I'll work on expanding the allowed characters. Thanks for reporting!
